landcover.mss

```diff
@@ -340,6 +341,7 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass'],
```

Comment:
I think there is a `[zoom >= 10]` missing here...

Reply:
I agree

Reply:
Indeed, fixed.